maplap/interface.py

```diff
@@ -0,0 +1,641 @@
+"""interface"""
+import sys
+
+from PyQt5 import Qt, QtCore, QtGui, QtWidgets
+
+import constant as C
+from design import UiMapLap
+
+
+class Rectangle:
+    """Rectangle struct"""
+
+    def __init__(self, x1=C.ZERO, y1=C.ZERO, x2=C.ZERO, y2=C.ZERO):
+        self.x_start = x1
+        self.y_start = y1
+        self.x_end = x2
+        self.y_end = y2
+
+    def none_for_pylint(self):
+        """do nothing"""
+        self.none_for_pylint()
+
+    def also_none_for_pylint(self):
+        """also do nothing"""
+        self.also_none_for_pylint()
+
+
+class SettingsParams:
+    """Params for settings"""
+
+    # pylint: disable=R1721
+    # I know that better - ok
+
+    def __init__(self, line_param):
+        params = [elem for elem in list(line_param.split(","))]
```

Comment:
Вроде бы это эквивалентно:
```suggestion
        params = list(line_param.split(","))
```